mixins/localize-dynamic-mixin.js

```diff
@@ -2,11 +2,17 @@ import { getLocalizeOverrideResources } from '../helpers/getLocalizeResources.js
 import { LocalizeMixin } from './localize-mixin.js';
 
 const fallbackLang = 'en';
+const supportedLangpacks = ['ar', 'cy', 'da', 'de', 'en', 'es', 'es-es', 'fi', 'fr', 'fr-fr', 'fr-on', 'hi', 'ja', 'ko', 'nl', 'pt', 'sv', 'tr', 'zh-cn', 'zh-tw'];
```

Comment:
Do we need `fi`?

Reply:
If My Courses (which actually has a `fi` file with real translations) ever moved to use the dynamic mixin, would that just mean you can't test `fi` locally (but in production it'd work fine)?

Reply:
Yeah exactly.

Reply:
I'm assuming my courses should probably remove their `fi`.

Reply:
Interesting. Feels weird to have `fi` support for some components/clients but not all.